pywb/rewrite/regex_rewriters.py

```diff
@@ -231,34 +270,12 @@ def final_read(self):
         return self.last_buff
 
 
-# =================================================================
-class JSLocationOnlyRewriter(JSLocationRewriterMixin, RegexRewriter):
-    pass
-
-
-# =================================================================
-class JSLinkOnlyRewriter(JSLinkRewriterMixin, RegexRewriter):
-    pass
-
-
-# =================================================================
-class JSLinkAndLocationRewriter(JSLocationRewriterMixin,
-                                JSLinkRewriterMixin,
-                                RegexRewriter):
-    pass
-
-
 # =================================================================
 class JSNoneRewriter(RegexRewriter):
     def __init__(self, rewriter, rules=[]):
```

Comment:
I think it maybe wise to change `rules=[]` to
```python
def __init__(self, rewriter, rules=None):
    if rules is None:
        rules = []
``` 
The default assignment to `[]` is a perf hit as the initial assignment persists across all creations and you never get a "new" list again.